go/types/list.go

```diff
@@ -116,15 +135,21 @@ func (l List) elemType() *Type {
 	return l.seq.Type().Desc.(CompoundDesc).ElemTypes[0]
 }
 
+// Set returns a new list where the valie at the given index have been replaced with v. If idx is
+// out bounds then this panics.
 func (l List) Set(idx uint64, v Value) List {
 	d.Chk.True(idx < l.Len())
```

Comment:
I can do it in another PR. I would prefer to keep this a documentation only change